pkg/mcproducer/templates/replace-kernel-module.gotmpl

```diff
@@ -2,14 +2,14 @@
 echo "before checking podman images"
 if podman image exists {{.Image}}; then
     echo "Image {{.Image}} found in the local registry, removing in-tree kernel module"
-    podman run --privileged --entrypoint modprobe {{.Image}} -rd /opt {{.KernelModule}}
+    podman run --user=root --privileged -v /etc/kmm-worker-day1/config.yaml:/etc/kmm-worker/config.yaml {{.Image}} kmod unload /etc/kmm-worker/config.yaml
```

Comment:
What is the benefits of using `kmod` rather than `modprobe`?

Reply:
this code actually runs worker pod, and kmod is part d the arguments to the worker app in the pod. modporbe will be called by the worker pod

Reply:
Ohh OK. This is confusing since there is a `kmod` binary loading kernel modules in Linux.